roles-in-tech/machine-learning.md

```diff
@@ -36,10 +39,35 @@
 
 ### Materials
 
+- [Example video (10 min)](https://example.com)
+- [Other example article(20 min read)](https://otherexample.com)
```

Comment:
Hi @rudri182 , please remove this link (https://otherexample.com), as it's an appropriate link. It will create a bad impact on users who'll click it(actually, it has some inappropriate content).